.claude/CLAUDE.md

```
@@ -0,0 +1,320 @@
+}
+```
+
+❌ **Don't fix unrelated bugs**
+- Focus on the task at hand
+- Note other issues in your final message
+- Create separate issues/PRs for unrelated fixes
+
+## Resources
+
+- Main guidelines: \`AGENTS.md\`
+- Repository: https://github.com/softmaple/softmaple
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)